### PR TITLE
More README changes

### PR DESCRIPTION
**Pull request by tphalp**
More iteration on repo readmes.

Resolves UCP-215

### CODE REVIEW

File: MISC.md [hunk 1]
```diff
@@ -28,10 +27,6 @@ docker compose up
 
 That's it! If you have questions, please reach out to us.
```
[the remark this answers is not in the document]
Comment (Craiting) [reply MISC.md:28]: You could link to this: https://universalconnectproject.org/build

File: MISC.md [hunk 2]
```diff
@@ -28,10 +27,6 @@ docker compose up
...
 
-## API Documentation
```
Comment (Craiting) [inline MISC.md:31]: Why are we removing this section?

Comment (tphalp) [reply MISC.md:31]: Relocated to the main README.

File: MISC.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tphalp) [reply MISC.md:28]: Thanks!

**Pull request merged**